frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.html

```diff
@@ -3,122 +3,130 @@
       <ngx-skeleton-loader
-        style="width: 100%"
-        *ngIf="
-          (!diagrams[diagram.uuid] && diagram.success) ||
-          diagrams[diagram.uuid]?.loading
-        "
+        *ngFor="let _ of loaderArray"
+        class="loader"
         [theme]="{
           'border-radius': '5px',
-          height: '200px',
-          width: 'calc(100% - 10px)',
+          height: '332px',
+          'min-width': '208px',
           border: '1px solid white'
         }"
       ></ngx-skeleton-loader>
-      <div *ngIf="!diagram.success" class="diagram-error">
-        <mat-icon class="diagram-error-icon">error</mat-icon> <br />
-        Diagram export has failed. <br />Please contact your diagram cache
-        administrator.
-      </div>
-      <div
-        *ngIf="
-          diagram.success &&
-          diagrams[diagram.uuid] &&
-          diagrams[diagram.uuid].content === null
-        "
-        class="diagram-error"
-      >
-        <mat-icon class="diagram-error-icon">error</mat-icon> <br />
-        Error loading the diagram from the server. <br />Please try again later.
-        <br />
-        If the problem persists, please contact your diagram cache
-        administrator.
-      </div>
-      <div class="list-item">
-        <div class="diagram-metadata">
-          <div class="diagram-name">{{ diagram.name }}</div>
-          <div class="uuid">UUID: {{ diagram.uuid }}</div>
-        </div>
-      </div>
-      <button
-        class="download-button"
-        (click)="downloadDiagram(diagram.uuid)"
-        mat-raised-button
-        [disabled]="!diagram.success"
-      >
-        <mat-icon>cloud_download</mat-icon> Download
-      </button>
-    </mat-card>
-  </div>
-  <div class="flex flex-wrap" *ngIf="filteredDiagrams() === undefined">
-    <ngx-skeleton-loader
-      *ngFor="let _ of loaderArray"
-      class="loader"
-      [theme]="{
-        'border-radius': '5px',
-        height: '332px',
-        'min-width': '208px',
-        border: '1px solid white'
-      }"
-    ></ngx-skeleton-loader>
-  </div>
-  <div *ngIf="this.diagramMetadata?.diagrams?.length === 0">
-    Your model doesn't seem to contain diagrams.
-  </div>
-  <div
-    *ngIf="
-      this.diagramMetadata?.diagrams?.length !== 0 &&
-      filteredDiagrams()?.length === 0
-    "
-  >
-    No diagrams for the given filter found. Please remove your search query.
-  </div>
-  <div class="sticky bottom-0 w-full bg-white md:hidden">
-    <mat-divider id="bottom-divider"></mat-divider>
-    <button class="w-full" mat-button mat-dialog-close>Close</button>
+    </div>
+    <div *ngIf="this.diagramMetadata?.diagrams?.length === 0">
+      Your model doesn't seem to contain diagrams.
+    </div>
+    <div
+      *ngIf="
+        this.diagramMetadata?.diagrams?.length !== 0 &&
+        filteredDiagrams()?.length === 0
+      "
+    >
+      No diagrams for the given filter found. Please remove your search query.
+    </div>
```

Comment:
This can be changed to the new angular syntax (i.e., using `@...`) and the first condition is no longer needed when using an `else if`

Reply:
I wouldn't change it as part of this PR as it involves some advanced testing (e.g. setting up a diagram cache without diagrams).